docs/integrations/sources/monday.md

```diff
@@ -57,6 +57,7 @@ The Monday connector should not run into Monday API limitations under normal usa
 
 | Version | Date       | Pull Request                                             | Subject                                         |
 |:--------|:-----------|:---------------------------------------------------------|:------------------------------------------------|
+| 0.2.1   | 2022-11-25 | [19586](https://github.com/airbytehq/airbyte/pull/19586) | Bump CDK version|
 | 0.2.0   | 2022-11-25 | [19586](https://github.com/airbytehq/airbyte/pull/19586) | Migrate to low-code          |
```

Comment:
Please update the date of your change as well as the last pr (with the actual merge date), and update pr # of your change

Reply:
done